changelog/1445.feature.rst

```diff
@@ -1 +1 @@
-Added is_response_retry variation of Retry.is_retry that considers `HTTPResponse`.
\ No newline at end of file
+Added `is_response_retry` variation of `Retry.is_retry` that considers `HTTPResponse`.
```

Comment:
```suggestion
Added ``is_response_retry`` variation of ``Retry.is_retry`` that considers ``HTTPResponse``.
```

reStructuredText doesn't use the same pre-formatted identifier as Markdown. It uses two backticks

Reply:
Thanks!